go-controller/pkg/ovn/egressip.go

```diff
@@ -94,6 +94,38 @@ func (oc *Controller) deleteEgressIP(eIP *egressipv1.EgressIP) error {
 	return nil
 }
 
+func (oc *Controller) isEgressNodeOffline(egressNode *kapi.Node) bool {
```

Comment:
the name makes it sound like it's just _checking_, not actually marking it offline. And like the other function ("is _back_ online") it would be good if the name made it clear that it's checking for a _change_ in the online/offline state, not just returning the current state.